components/feature/syncedtabs/src/main/java/mozilla/components/feature/syncedtabs/SyncedTabsStorageSuggestionProvider.kt

```diff
@@ -16,10 +17,14 @@ import java.util.UUID
  * A [AwesomeBar.SuggestionProvider] implementation that provides suggestions for remote tabs
  * based on [SyncedTabsStorage].
  */
+@Suppress("LongParameterList")
 class SyncedTabsStorageSuggestionProvider(
     private val syncedTabs: SyncedTabsStorage,
     private val loadUrlUseCase: SessionUseCases.LoadUrlUseCase,
-    private val icons: BrowserIcons? = null
+    private val icons: BrowserIcons? = null,
+    private val indicatorDesktop: Drawable? = null,
+    private val indicatorMobile: Drawable? = null,
+    private val indicatorTablet: Drawable? = null
```

Comment:
```suggestion
    private val icons: BrowserIcons? = null,
    private val deviceIndicators: DeviceIndicators = DeviceIndicators()
```

How about we turn this into a data class of indicators instead?

```kotlin
data class DeviceIndicators(
    val desktop: Drawable? = null,
    val mobile: Drawable? = null,
    val tablet: Drawable? = null,
)
```

Reply:
Ohh i didn't know we can do like that. Great!
I'm not good with tests but i'll try my best.

Reply:
Have a look at other tests in the codebase, and how `SyncedTabsStorageSuggestionProviderTest` can also help give some ideas on how to create more tests there.

Feel free to ping me if you need more help! :)